### PR TITLE
Move class-transformer call to improve performance

### PR DESCRIPTION
**Pull request by paulhowes-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-2741

### Change description ###
Performance issue which impacts Employment Tribunals usage of Manage Cases. Issue is clearly apparent when loading a case with large hearing and judgement collections.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## RELEASE NOTES
 
+### Version 2.65.7
```
Comment (Daniel-Lam) [inline RELEASE-NOTES.md:3]: @paulhowes-HMCTS - there appears to be a mismatch in version numbers... you've got `2.65.7` here but `2.66.0` in `package.json`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "2.65.6",
+  "version": "2.66.0-performance-issue-tribunals",
```
Comment (Daniel-Lam) [inline package.json:3]: @paulhowes-HMCTS - see comment above re apparent mismatching version numbers.

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@paulhowes-HMCTS - just a version number inconsistency to look at, cheers

File: RELEASE-NOTES.md (hunk 1 above)
Comment (paulhowes-HMCTS) [reply RELEASE-NOTES.md]: I've pushed a new version of the release notes up now.

**Review by Daniel-Lam — APPROVED** [review]
Approved, @paulhowes-HMCTS  - version numbers now aligned 👍

**Review by paul-graham — APPROVED** [review]
Everything appears to be in order. 👍

**Pull request merged**